tests/integration/users/test_pending_users.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestPendingUsers:
+    def test_users_pending(self, client, valid_header, create_unauthorized_user):
+        pending_user = create_unauthorized_user()
+        assert pending_user.role is None
+        assert pending_user.archived is False
+
+        """The get pending users returns one user with no role
+        and a successful response code."""
+        response = client.get(
+            "/api/users/pending",
+            headers=valid_header,
+        )
+        assert len(response.get_json()["users"]) == 1
```

Comment:
What we want to test here is that we get back the expected json response, not the length of the response. We want to assert the actual response so that I can be confident that we are returning the correct data to the frontend.